dtfs-central-api/api-tests/v1/utilisation-reports/fee-record-corrections/put-fee-record-correction-transient-form-data.api-test.ts

```diff
@@ -122,14 +146,46 @@ describe(`PUT ${BASE_URL}`, () => {
     expect(status).toEqual(HttpStatusCode.NotFound);
   });
 
-  it(`should return '${HttpStatusCode.Ok}' if the correction exists`, async () => {
-    // Arrange
-    const requestBody = aValidRequestBody();
+  describe('when there are validation errors', () => {
+    it(`should return '${HttpStatusCode.Ok}' with the errors in the response body`, async () => {
+      // Arrange
+      const formData = {
+        utilisation: 'invalid-utilisation',
+        facilityId: '77777777',
+        reportedCurrency: 'invalid-currency',
+        reportedFee: 'invalid-reported-fee',
+        additionalComments: '',
+      };
+
+      const requestBody = {
+        ...aValidRequestBody(),
+        formData,
+      };
+
+      // Act
+      const { status, body } = (await testApi
+        .put(requestBody)
+        .to(replaceUrlParameterPlaceholders(BASE_URL, { bankId, correctionId }))) as PutFeeRecordCorrectionTransientFormDataResponse;
+
+      // Assert
+      const { errors: expectedBody } = await validateRecordCorrectionTransientFormValues(formData, correctionReasons);
```

Comment:
Given api-tests are somewhat intentionally integration tests, I would prefer we didn't call this here to get the expected body